justfile

```diff
@@ -120,6 +123,10 @@ check *args: devenv
     $BIN/black --check .
     $BIN/isort --check-only --diff .
     $BIN/flake8
+    $BIN/pyupgrade --py311-plus \
+    $(find builder codelists conversions mappings opencodelists services superusers  -name '*.py' -type f) \
+    $(find coding_systems -not -path 'coding_systems/snomedct/parser_utils/*' -name '*.py' -type f)
+
```

Comment:
suggestion: It might be useful to comment why these files are selected and excluded in this way.